assets/prototype/domain/eventEditor/containers/mutations/prices/usePriceMutator.js

```diff
@@ -0,0 +1,61 @@
+import usePriceQueryOptions from './usePriceQueryOptions';
+import useOnCreatePrice from './useOnCreatePrice';
+import useOnUpdatePrice from './useOnUpdatePrice';
+import useOnDeletePrice from './useOnDeletePrice';
+
+/**
+ *
+ */
+const usePriceMutator = () => {
+	const options = usePriceQueryOptions();
+
+	const onCreatePrice = useOnCreatePrice();
+	const onUpdatePrice = useOnUpdatePrice();
+	const onDeletePrice = useOnDeletePrice();
+
+	const createVariables = (mutationType, input) => {
+		const mutationInput = {
+			clientMutationId: `${mutationType}_PRICE`,
+			...input,
+		};
+
+		return {
+			input: mutationInput,
+		};
+	};
+
+	const mutator = (mutationType, input) => {
+		// `ticketId` must be passed for price creation
+		// so as to properly set the relations
+		const { ticketId, ...restInput } = input;
+		const { priceType: priceTypeId } = input;
```

Comment:
What do you think about this:
```suggestion
		const { ticketId, priceType: priceTypeId, ...restInput } = input;
```
?

Reply:
`priceType` is needed in `restInput`

Reply:
Or maybe use `omit` from ramda:
https://ramdajs.com/docs/#omit
in `createVariables`

Reply:
I understood you didn't want to pass entire `input` object.

Reply:
`ticketId` is still to be removed as it's not a part of the mutation schema 😄